### PR TITLE
sequence multi retrieve

### PR DESCRIPTION
**Pull request by sanderland**
Based on statnett needs

### CODE REVIEW

**Comment by sanderland** [comment]
> LGTM. But maybe the `column_names` argument should include the curly brackets? I think it would be easier to understand how the argument works. Also I think they should be snake_cased since that is how they pass the arguments elsewhere. So instead of passing for example "externalId|columnExternalId" you would pass "{external_id}|{column_external_id}".

I was trying to keep it more similar to what we do for datapoints, where we do snake case the parameters. But the templating has no precedent really, so I'm not sure there, will ask around a bit.

**Comment by erlendvollset** [comment]
With my example it would work like the standard f-string, which seems natural to me.

**Comment by sanderland** [comment]
> With my example it would work like the standard f-string, which seems natural to me.

Yes, I got that, and I think it's a good idea if we didn't have anything yet, but it would also lead to:

`seq_data.to_pandas(column_names='{external_id}')` being different and not a natural extension from the already existing `dpt.to_pandas(column_names='externalId')`

**Comment by erlendvollset** [comment]
Right. Maybe just stick to the way it is done there and not support templating here then. Do we have requests for multiple identifiers in the same column header?

**Comment by sanderland** [comment]
f

> Right. Maybe just stick to the way it is done there and not support templating here then. Do we have requests for multiple identifiers in the same column header?

It's hard to make them unique otherwise. We need to make them columnExernalId for backward compatibility if 1 sequence is used, but multi sequence single column retrieval is the use case that led to this PR, where (sequence) externalId would be natural. Without assumptions and supporting both you're led to this kind of combination template.

File: cognite/client/data_classes/sequences.py [hunk 1]
```diff
@@ -276,21 +276,42 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
...
+        if column_names not in [
+            "externalId",
+            "id",
+            "columnExternalId",
+            "id|columnExternalId",
+            "externalId|columnExternalId",
+        ]:
+            raise ValueError(
+                'Invalid column_names value, should be one of "externalId", "id", "columnExternalId", "id|columnExternalId", "externalId|columnExternalId"'
+            )
```
Comment (erlendvollset) [inline cognite/client/data_classes/sequences.py]:
```
options = [
            "externalId",
            "id",
            "columnExternalId",
            "id|columnExternalId",
            "externalId|columnExternalId",
        ]
if column_names not in options:
    raise ValueError(f"Invalid column_names value, should be one of {options}")
```

Comment (sanderland) [reply cognite/client/data_classes/sequences.py]: fixed in a py3.5 compatible way (although using f-strings would be awesome)

**Pull request merged**